File: Madrassaty.Back/QuranRecitation.WebApi/Models/RecitationSessionModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace QuranRecitation.WebApi.Models
{
    public class RecitationSessionModel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public bool IsSaved { get; set; }
        public Guid TeacherId { get; set; }
        public int ClassroomId { get; set; }
        public string Description { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndTime { get; set; }
        public int DivisionParam { get; set; }
        public byte[] CreatedOn { get; set; }
        public DateTime ModifiedOn { get; set; }
        public int? RecurrenceId { get; set; }
        public int DisciplineId { get; set; }
        public string TypeEvaluation { get; set; }
        public string Jour { get; set; }
        public DateTime? T1 { get; set; }
        public DateTime? T2 { get; set; }
        public DateTime? T3 { get; set; }
        
        public ICollection<Guid> IdStudents { get; set; }
    }
}

File: Madrasaty.Front/src/app/models/recitation-session-model.ts
export class RecitationSessionModel {
  Id: number;
  Title: string;
  IsSaved: boolean;
  TeacherId: string;
  ClassroomId: number;
  Description: string;
  StartDate: Date;
  EndTime: Date;
  DivisionParam: number;
  CreatedOn: Date;
  ModifiedOn: Date;
  RecurrenceId: number;
  DisciplineId: number;
  IdStudents: string[];
  TypeEvaluation: string;
  Jour: string;
  T1: Date;
  T2: Date;
  T3: Date;
}
